### PR TITLE
chore!(base-driver,appium): remove support for execute driver script

### PR DESCRIPTION
**Pull request by jlipps**
This is being moved to a plugin: https://github.com/appium/appium-plugins/pull/73

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

File: packages/base-driver/package.json [hunk 1]
```diff
@@ -57,7 +57,6 @@
     "serve-favicon": "2.5.0",
     "source-map-support": "0.5.21",
     "validate.js": "0.13.1",
-    "webdriverio": "6.12.1",
```
Comment (mykola-mokhnach) [inline packages/base-driver/package.json:60]: 🎉

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by boneskull** [comment]
What is the `!` syntax in the commit msg?

File: packages/appium/lib/appium.js [hunk 2]
```diff
@@ -216,6 +216,12 @@ class AppiumDriver extends BaseDriver {
 
       // This assignment is required for correct web sockets functionality inside the driver
       driverInstance.server = this.server;
+
+      // Drivers/plugins might also want to know where they are hosted
+      driverInstance.serverHost = this.args.address;
```
Comment (boneskull) [inline packages/appium/lib/appium.js:221]: maybe we should take this opportunity to add a setter or setters to the driver class for this stuff.  at the very least it'd be less of a mystery where these properties originate

File: packages/base-driver/test/basedriver/driver-e2e-tests.js [hunk 3]
```diff
@@ -1,8 +1,5 @@
 import _ from 'lodash';
-import { BaseDriver, server, routeConfiguringFunction, DeviceSettings, errors } from '../../lib';
-import {
-  MJSONWP_ELEMENT_KEY, W3C_ELEMENT_KEY
-} from '../../lib/constants';
+import { BaseDriver, server, routeConfiguringFunction, DeviceSettings } from '../../lib';
```
Comment (boneskull) [inline packages/base-driver/test/basedriver/driver-e2e-tests.js:2]: can we remove stuff from `lib/constants` too?

**Review by boneskull — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
> What is the ! syntax in the commit msg?

Means the commit contains breaking changes: https://www.conventionalcommits.org/en/v1.0.0/

File: packages/appium/lib/appium.js (hunk 2 above)
Comment (jlipps) [reply packages/appium/lib/appium.js:221]: https://github.com/appium/appium/issues/16156

File: packages/base-driver/test/basedriver/driver-e2e-tests.js (hunk 3 above)
Comment (jlipps) [reply packages/base-driver/test/basedriver/driver-e2e-tests.js:2]: they are used elsewhere. but it's ok, they just import the value from appium support in any case.

**Pull request merged**